### PR TITLE
Electron keyboard shortcuts tests

### PR DESCRIPTION
**Pull request by nabil-brn**
### 📝 Description

Some regression tests about keyboard shortcuts
https://ledger.slack.com/archives/C02E0PU4UV8/p1676029549759949

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+test.use({ userdata: "skip-onboarding" });
+
+// eslint-disable-next-line jest/expect-expect
+test("Keyboard shortcuts", async ({ page }) => {
+  const layout = new Layout(page);
+  const portfolioPage = new PortfolioPage(page);
+  const addAccountModal = new AddAccountModal(page);
+
+  // test that CTRL+SHIT+I doesn't open devtools
```
Comment (elGrogz) [inline apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts:16]: image

File: apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import os from "os";
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { Layout } from "../../models/Layout";
+import { AddAccountModal } from "tests/models/AddAccountModal";
+import { PortfolioPage } from "tests/models/PortfolioPage";
+
+test.use({ userdata: "skip-onboarding" });
+
+// eslint-disable-next-line jest/expect-expect
```
Comment (abdurrahman-ledger) [inline apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts:10]: Is this really needed ?

**Review by abdurrahman-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/tests/specs/general/devmode.spec.ts [hunk 3]
```diff
@@ -1,61 +0,0 @@
...
-      await deviceAction.openApp();
-      await addAccountModal.waitForSync();
-      expect
-        .soft(await addAccountModal.container.screenshot())
-        .toMatchSnapshot(`scan-${currency}-accounts.png`);
-      await addAccountModal.addAccounts();
-      await addAccountModal.done();
-    });
-  }
-});
```
Comment (abdurrahman-ledger) [inline apps/ledger-live-desktop/tests/specs/general/devmode.spec.ts:61]: Not tested anymore ?

Comment (nabil-brn) [reply apps/ledger-live-desktop/tests/specs/general/devmode.spec.ts:61]: Nope, it was outdated and skipped, also covering dev mode is not a priority

File: apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts (hunk 2 above)
Comment (nabil-brn) [reply apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts:10]: Yes I test keyboard navigation in modals, so I need to skip onboarding.

**Pull request merged**